### PR TITLE
React 15.6.1

### PR DESCRIPTION
**Pull request by evilj0e**
* Update deps to green
* Make fixes for React 15.6.1

### CODE REVIEW

File: src/components/Link.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import BaseRouterComponent from './Base';
 
-import shallowCompare from 'react-addons-shallow-compare';
```
Comment (TimeRaider) [inline src/components/Link.js]: should probably remove from package.json as well

**Comment by evilj0e** [comment]
@TimeRaider, have a look.

**Comment by evilj0e** [comment]
Feat #43, #44, #45, #46, #47

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,6 @@
 language: node_js
 node_js:
```
Comment (TimeRaider) [inline .travis.yml]: lets add `7` and `8` as well

Comment (TimeRaider) [inline .travis.yml]: can do later

**Pull request merged**